cmd/log-generator/log-generator.go

```diff
@@ -25,7 +23,7 @@ var (
 	eventSize       = flag.Int("eventSize", 120, "Identify the single log line size, in Byte.")
 	outPutPath      = flag.String("path", "", "Identify the path where log files will generate.")
 	filePrefix      = flag.String("filePrefix", "tmp", "Identify the log file prefix")
-	loopNum         = flag.Int64("loopNum", math.MaxInt64, "How many loops to write teh logentry.")
```

Comment:
For context these programs were just copied from an internal test repository. However `loopNum` is not needed. It indirectly controls the runtime, so I replaced it.

Reply:
Nice, this is much cleaner